src/main/java/co/infinum/complexify/Complexify.java

```diff
@@ -188,14 +190,15 @@ public Complexify() {
     /**
      * Constructor
      *
-     * @param banMode             Use strict or loose comparisons for banned passwords. (default: ComplexifyBnaMode.STRICT)
+     * @param banMode             Use strict or loose comparisons for banned passwords. (default: ComplexifyBanMode.STRICT)
      * @param strengthScaleFactor Required password strength multiplier (default: 1)
      * @param minimumChars        Minimum password length (default: 8)
      */
     public Complexify(ComplexifyBanMode banMode, int strengthScaleFactor, int minimumChars) {
         this.banMode = banMode;
```

Comment:
I agree. Leave it as is for now and fix it in a different PR. I do not have a problem with bumping major version once we add builder pattern.